packages/amplify-migration-e2e/src/migrationReadmeParser.ts

```diff
@@ -0,0 +1,35 @@
+function extractContent(readmeContent: string, startRegex: string, endRegex: string) {
+  const pattern = new RegExp(`${startRegex}([\\s\\S]*?)${endRegex}`, 'i');
+  const match = readmeContent.match(pattern);
+
+  if (match && match[1]) {
+    return match[1].trim();
+  }
+  throw new Error('README file parsing failed to get the stack refactor commands');
+}
+
+function extractCommands(readmeContent: string) {
+  const pattern = /```([\s\S]*?)```/g;
+  const matches = readmeContent.matchAll(pattern);
+  const commands = [];
+
+  for (const match of matches) {
+    if (match[1]) {
+      commands.push(match[1].trim());
+    }
+  }
+  if (commands.length === 0) {
+    throw new Error('README file parsing failed to get the stack refactor commands');
+  }
+  return commands;
+}
+
+export function getCommandsFromReadme(readmeContent: string) {
```

Comment:
Referencing the comment from the previous PR: https://github.com/aws-amplify/amplify-cli/pull/13984#discussion_r1809384187

Reply:
nit: give some sample of the readme fie, like the step 1 portion, would be helpful to visualize the file format.